vhsdecode/process.py

```diff
@@ -12,6 +12,19 @@
 
 import vhsdecode.formats as vhs_formats
 from vhsdecode.addons.FMdeemph import FMDeEmphasis
+from collections import deque
+from fractions import Fraction
+from samplerate import resample
```

Comment:
Sure, but it seems to be slower.
I corrected it in a new commit.
It picks samplerate if available, if it is not, it prints a warning and uses np.interp() for linear resampling.
If the resampling method is not linear, it uses signal.resample() (takes forever)

I fixed some threading issues that I introduced with VTRDemodCache() worker override
And added the trap over the cvbs luminance information.

I need to figure out how to adjust the EQ if the trap is disabled because disabling it will cause the EQ to overboost the high frequencies.